src/lib_ccx/ccx_encoders_common.c

```diff
@@ -55,47 +55,47 @@ ScriptType: v4.00+\n\
 \n\
 [Events]\n\
 Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text\n\
 \n";
 
-/*	
+/*
 
 TODO -- Set correct values for "ttp:dropMode", "ttp:frameRate" .Using common for now.
 
- Extra reference, 
+ Extra reference,
 
 	24 frame/sec (film, ATSC, 2k, 4k, 6k)
 	25 frame/sec (PAL (Europe, Uruguay, Argentina, Australia), SECAM, DVB, ATSC)
-	29.97 (30 ÷ 1.001) frame/sec (NTSC American System (US, Canada, Mexico, Colombia, etc.), ATSC, PAL-M (Brazil))
+	29.97 (30 รท 1.001) frame/sec (NTSC American System (US, Canada, Mexico, Colombia, etc.), ATSC, PAL-M (Brazil))
 	30 frame/sec (ATSC)
-	
+
 	Find framerate, then use it find dropmode.
-	
+
 */
 static const char *smptett_header = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\" ?>\n"
 			"  <tt xmlns=\"http://www.w3.org/ns/ttml\" xmlns:ttp=\"http://www.w3.org/ns/ttml#parameter\" ttp:dropMode=\"dropNTSC\" ttp:frameRate=\"30\" ttp:frameRateMultiplier=\"1000 1001\" ttp:timeBase=\"smpte\" xmlns:m608=\"http://www.smpte-ra.org/schemas/2052-1/2010/smpte-tt#cea608\" xmlns:smpte=\"http://www.smpte-ra.org/schemas/2052-1/2010/smpte-tt\" xmlns:ttm=\"http://www.w3.org/ns/ttml#metadata\" xmlns:tts=\"http://www.w3.org/ns/ttml#styling\">\n"
 //			"  <tt xmlns=\"http://www.w3.org/ns/ttml\" xmlns:ttp=\"http://www.w3.org/ns/ttml#parameter\" xmlns:m608=\"http://www.smpte-ra.org/schemas/2052-1/2010/smpte-tt#cea608\" xmlns:smpte=\"http://www.smpte-ra.org/schemas/2052-1/2010/smpte-tt\" xmlns:ttm=\"http://www.w3.org/ns/ttml#metadata\" xmlns:tts=\"http://www.w3.org/ns/ttml#styling\">\n"
 			"  <head>\n"
-			"    <styling>\n"
-			"      <style tts:color=\"white\" tts:fontFamily=\"monospace\" tts:fontWeight=\"normal\" tts:textAlign=\"left\" xml:id=\"basic\"/>\n"
-			"    </styling>\n"
-			"    <layout>\n"
-			"      <region tts:backgroundColor=\"transparent\" xml:id=\"pop1\"/>\n"
-			"      <region tts:backgroundColor=\"transparent\" xml:id=\"paint\"/>\n"
-			"      <region tts:backgroundColor=\"transparent\" xml:id=\"rollup2\"/>\n"
-			"      <region tts:backgroundColor=\"transparent\" xml:id=\"rollup3\"/>\n"
-			"      <region tts:backgroundColor=\"transparent\" xml:id=\"rollup4\"/>\n"
-			"    </layout>\n"
-			"    <metadata/>\n"
-			"    <smpte:information m608:captionService=\"F1C1CC\" m608:channel=\"cc1\"/>\n"
+			"	<styling>\n"
+			"	<style tts:color=\"white\" tts:fontFamily=\"monospace\" tts:fontWeight=\"normal\" tts:textAlign=\"left\" xml:id=\"basic\"/>\n"
+			"	</styling>\n"
+			"	<layout>\n"
+			"	<region tts:backgroundColor=\"transparent\" xml:id=\"pop1\"/>\n"
+			"	<region tts:backgroundColor=\"transparent\" xml:id=\"paint\"/>\n"
+			"	<region tts:backgroundColor=\"transparent\" xml:id=\"rollup2\"/>\n"
+			"	<region tts:backgroundColor=\"transparent\" xml:id=\"rollup3\"/>\n"
+			"	<region tts:backgroundColor=\"transparent\" xml:id=\"rollup4\"/>\n"
+			"	</layout>\n"
+			"	<metadata/>\n"
+			"	<smpte:information m608:captionService=\"F1C1CC\" m608:channel=\"cc1\"/>\n"
```

Comment:
All the changes there are wrong - you are modifying the string literals used for a header... these need to be left alone.